keyboards/bastardkb/charybdis/3x6/3x6.h

```diff
@@ -20,7 +20,7 @@
 #include "charybdis.h"
 
 // clang-format off
-#define LAYOUT_charybdis_3x6(                                     \
+#define LAYOUT(                                                   \
```

Comment:
Needs alias similar to the others.

Reply:
Does it need a _de facto_ alias (in case it's used on keymaps that are not submitted, maybe?).

It's currently not used by any of the published keymap.

Reply:
For out of repo keymaps. Otherwise it has to go via `develop` which will merge end of Feb.

Reply:
Fair enough, thanks for the explanation. Done in the last commit.